app/src/org/commcare/views/widgets/WidgetFactory.java

```diff
@@ -61,7 +61,11 @@ public QuestionWidget createWidgetFromPrompt(FormEntryPrompt fep, Context contex
                             questionWidget = new EthiopianDateWidget(context, fep);
                         } else if (appearance != null && appearance.toLowerCase().equals("nepali")) {
                             questionWidget = new NepaliDateWidget(context, fep);
-                        } else {
+                        }else if(appearance != null && appearance.toLowerCase().equals("greg")){ //TESTING GREGORIAN WIDGET
+                            questionWidget = new GregorianWidget(context, fep);
+                        }else if(appearance != null && appearance.toLowerCase().equals("calendar")){
+                            questionWidget = new CalendarWidget(context, fep);
```

Comment:
They're completely unrelated - GregorianWidget is a Gregorian counterpart to Nepali and Ethiopian...all 3 of these inherit from the same superclass. 

CalendarWidget's GUI and internal logic are totally different. Will rename these classes to be more clear.